source/_integrations/homekit.markdown

```diff
@@ -21,7 +21,7 @@ The HomeKit Bridge integration allows you to forward entities from Home Assistan
 
 <div class="note warning">
 
-If you are using the official Home Assistant images or running Home Assistant Core on Docker, HomeKit is ready to go out of the box. If you are running Home Assistant in a manual virtual environment or on a NAS without Docker, you may need to install or upgrade dependencies for HomeKit to function.
+If you are using the official Home Assistant images or running Home Assistant Container, HomeKit is ready to go out of the box. If you are running Home Assistant in a manual virtual environment or on a NAS without Docker, you may need to install or upgrade dependencies for HomeKit to function.
```

Comment:
This sentence itself is uh, weird. We always need to install dependencies to use integrations. Can we just remove this paragraph ? 

The other stuff also doesn't belong in a warning. 

(ok, out of scope for this PR)

Reply:
Removed.